### PR TITLE
utftex 1.17 (new formula)

### PR DESCRIPTION
**Pull request by larseggert**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

### CODE REVIEW

**Comment by chenrui333** [comment]
> I get `Libraries were compiled with a flat namespace.` but no idea how to fix?

check out this commit, https://github.com/Homebrew/formula-patches/commit/03cf8088210822aa2c1ab, we have many formulae in the codebase of using that commit for solving the libtool flat namespace issue.

**Comment by larseggert** [comment]
@chenrui333 thanks! I ended up rerunning `autoconf`/`automake` since the packaged `configure` script was too old for the patch to apply cleanly.

**Comment by larseggert** [comment]
Although the `utftex` repo may not be notable enough, `utftex` is now part of the main IETF toolchain to generate documents, i.e., https://github.com/cabo/kramdown-rfc, which should bring it across the bar?

File: Formula/utftex.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  sha256 "a1ecfe44f396c77565515ee1d93207e794d3b99f3423fd2b8e32d53ef59c9d50"
+  license "GPL-3.0-only"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+
+  def install
+    system "./autogen.sh"
+    system "./configure", *std_configure_args, "--disable-silent-rules", "--prefix=#{prefix}"
```
Comment (SMillerDev) [inline Formula/utftex.rb:14]: This is in the standard arguments
```suggestion
    system "./configure", *std_configure_args, "--disable-silent-rules"
```

**Review by SMillerDev — DISMISSED** [review]

File: Formula/utftex.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  depends_on "libtool" => :build
+
+  def install
+    system "./autogen.sh"
+    system "./configure", *std_configure_args, "--disable-silent-rules"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/utftex '(\\frac{hello}{world})'"
```
Comment (SMillerDev) [inline Formula/utftex.rb:19]:
```suggestion
    system bin/"utftex", "'(\\frac{hello}{world})'"
```

**Comment by SMillerDev** [comment]
Can you remove the merge commit? Our CI can't merge with merge commits

**Comment by larseggert** [comment]
@SMillerDev OK now?

File: Formula/utftex.rb (hunk 2 above)
Comment (SMillerDev) [reply Formula/utftex.rb:19]: Did this not work @larseggert ?

Comment (larseggert) [reply Formula/utftex.rb:19]: Sorry. I must have reverted this by accident. Will make the change again.

**Review by SMillerDev — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@larseggert, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**